scootapi/setup/setup.go

```diff
@@ -1,3 +1,6 @@
+// Package setup provides Cloud Scoot Setup, which is a mechanism for
```

Comment:
It would also help if we documented some of our conventions. Like flesh this out a little bit and make it applicable to the other components that do setup type stuff.